config/zones/LT.yaml

```diff
@@ -110,6 +115,11 @@ capacity:
     - datetime: '2020-01-01'
       source: entsoe.eu
       value: 144.0
+    - datetime: '2024-10-22'
+      # https://www.litgrid.eu/index.php/energetikos-sistema/elektros-energetikos-sistemos-informacija/irengtoji-galia/502
+      # 6394 - (338 + 1166 + 900 + 1669 + 1878 + 129 + 198 + 10)
+      _comment: Waste included (69 MW)
```

Comment:
Waste should probably be mapped to biomass since it's mapped to that in the parser.